doc/rbd/rados-rbd-cmds.rst

```diff
@@ -65,24 +65,30 @@ Creating a Block Device Image
 =============================
 
 Before you can add a block device to a node, you must create an image for it in
-the :term:`Ceph Storage Cluster` first. To create a block device image, execute
-the  following::
+the :term:`Ceph Storage Cluster`. To create a block device image, run a command of this form: 
 
-	rbd create --size {megabytes} {pool-name}/{image-name}
+.. prompt:: bash $
+
+   rbd create --size {megabytes} {pool-name}/{image-name}
 
 For example, to create a 1GB image named ``bar`` that stores information in a
-pool named ``swimmingpool``, execute the following::
+pool named ``swimmingpool``, run this command:
+
+.. prompt:: bash $
+
+   rbd create --size 1024 swimmingpool/bar
 
-	rbd create --size 1024 swimmingpool/bar
+If you don't specify a pool when you create an image, then the image will be
+stored in the default pool ``rbd``. For example, if you ran this command, you
```

Comment:
ran/will?

Reply:
The "ran" is just past subjunctive. I suppose I could write it "if you were to run this command" or "were you to run this command", but those seem prolix to me.

I toyed with "is" in the place of "will be", which would have made the sentence "If you don't specify a pool when you create an image, then the image is stored in the default pool 'rbd'.". The clause "will be stored in the default pool" was chosen because the sentence's main clause is about the creation of an image, an act that occurs before the storage of that image. However, upon the review that your question has occasioned, I might well return to recast this sentence in the present tense.